src/System.IdentityModel.Tokens.Jwt/ClaimsAccessorsDoc.md

```diff
@@ -0,0 +1,85 @@
+﻿
+# Claims
+
+Type | Accessor | Used for authorization | Privacy | Claims
+--   | --       | --                     | --      | --     
+string | GetActor | False |  | JwtRegisteredClaimNames.Actort<BR/> ClaimTypes.Actor
+string | GetBirthdate | False | EUII | JwtRegisteredClaimNames.Birthdate<BR/> ClaimTypes.DateOfBirth
+string | GetEmail | False | EUII | JwtRegisteredClaimNames.Email<BR/> ClaimTypes.Email
+string | GetFamilyName | False | EUII | JwtRegisteredClaimNames.FamilyName<BR/> ClaimTypes.Surname
+string | GetGender | False | EUII | JwtRegisteredClaimNames.Gender<BR/> ClaimTypes.Gender
+string | GetGivenName | False | EUII | JwtRegisteredClaimNames.GivenName<BR/> ClaimTypes.GivenName
+string | GetNameid | False |  | JwtRegisteredClaimNames.NameId<BR/> ClaimTypes.NameIdentifier
+string | GetSub | False |  | JwtRegisteredClaimNames.Sub<BR/> ClaimTypes.NameIdentifier
+string | GetWebsite | False |  | JwtRegisteredClaimNames.Website<BR/> ClaimTypes.Webpage
+string | GetUniqueName | False |  | JwtRegisteredClaimNames.UniqueName<BR/> ClaimTypes.Name
+string | GetObjectId | True | EUPI | "oid"<BR/> "http://schemas.microsoft.com/identity/claims/objectidentifier"
+IEnumerable<string> | GetScopes | True |  | "scp"<BR/> "http://schemas.microsoft.com/identity/claims/scope"
+string | GetTenantId | True | OII | "tid"<BR/> "http://schemas.microsoft.com/identity/claims/tenantid"
+string | GetAcr | False |  | "acr"<BR/> "http://schemas.microsoft.com/claims/authnclassreference"
+string | GetAdfs1Email | False | EUPI | "adfs1email"<BR/> "http://schemas.xmlsoap.org/claims/EmailAddress"
+string | GetAdfs1Upn | False | EUPI | "adfs1upn"<BR/> "http://schemas.xmlsoap.org/claims/UPN"
```

Comment:
>GetAdfs1Upn [](http://example.com/codeflow?start=9&length=11)

Wouldn't this cause confusion as it references a claim like upn which will almost always be present in tokens issued to users?
It might give n incorrect impression that the user authenticated with ADFS, even if they didn't.